codejail_service/tests/test_startup_check.py

```diff
@@ -161,6 +162,20 @@ def test_egress_success_and_failure(self, mock_args, expected_status):
         assert startup_check.STARTUP_SAFETY_CHECK_OK is expected_status
         mock_urlopen.assert_called_once()
 
+        # Piggy-backing on this test to look at our monitoring calls as well
+        passfail = 'pass' if expected_status else 'fail'
+        assert mock_set_custom_attribute.call_args_list == [
+            # The other checks
+            call('codejail.startup_check.functionality', 'pass'),
+            call('codejail.startup_check.disk', 'pass'),
+            call('codejail.startup_check.exec', 'pass'),
+            call('codejail.startup_check.network', 'pass'),
+            # This check
+            call('codejail.startup_check.webapp_egress', passfail),
```

Comment:
Silly, but `passfail` tripped me up. What about `expected_status_as_string`, or something like this?

Reply:
Changed to `expected_status_attr_value`.

The distinction between True/False (or actually, True/anything-else) and pass/fail also tripped me up for a while during development and testing.